test/scenarios/4. tokens/approve erc721 token.csv

```diff
@@ -0,0 +1,21 @@
+Step,Test steps,Test data,Expected result,Notes
+1,Open the extension.,,The Welcome Back screen is shown.,
+2,Proceed to Unlock the wallet.,password (8 characters min).,"The Ether balance is shown on the overview. The wallet address is shown on the overview. The selected network is Ethereum Mainnet.        ",
+3,Switch networks to a test network.,e.g. Sepolia,The Sepolia balance is shown on the overview. The wallet address is shown on the overview. The selected network is Sepolia.,
+4,Open the test dapp in another tab.,https://metamask.github.io/test-dapp/,,
+5,"Proceed to connect with MetaMask. ",,,
+6,"Connect with the current account. ",,,
+7,"Deploy the NFT contract. ",,,
+8,Confirm the transaction and wait for it to complete.,,,
+9,Switch to the wallet and open the activity list item.,,"The contract deployment transaction status, nonce, amount, gas and total are shown in the item details.",
+10,"Proceed to mint NFT. ",,,
+11,Confirm the transaction and wait for it to complete.,,,
+12,Switch to the wallet and open the activity list item.,,"The mint NFT transaction status, recipient's address, nonce, amount, gas and total are shown in the item details.",
+13,Proceed to view the transaction on the block explorer.,,The block explorer opens in a new tab.,
+14,"Switch to test dapp, in the input field of approve NFT token section, enter the NFT ID you wish to approve.",e.g. 3,,"If NFT with token ID #3 should be appproved, amount of NFTs minted should be equal to or greater than 3."
+15,"In the test dapp, click ""Approve"".",,"Notification window is shown, asking user to allow access to the minted NFT.",
+16,Proceed to confirm the NFT approval.,,,
+17,Switch back to the wallet.,,The approve NFT spending cap transaction appears in the activity list. The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.,
```

Comment:
> The recipient's hexadecimal address is shown in the activity list item. The amount is shown in the activity list item.

This is not shown on Approve spending cap list item

Reply:
Nice catch, thanks !  Corrected.